Example/Stripe iOS Example (Simple).xcworkspace/contents.xcworkspacedata

```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this workspace in the Example directory was broken, and not useful, so I removed it.